dev/com.ibm.ws.clientcontainer.json_fat/fat/src/com/ibm/ws/clientcontainer/fat/FATSuite.java

```diff
@@ -0,0 +1,49 @@
+/*******************************************************************************
+ * Copyright (c) 2015,2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.clientcontainer.fat;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.EnterpriseArchive;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.BeforeClass;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+import org.junit.runners.Suite.SuiteClasses;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+
+import componenttest.custom.junit.runner.AlwaysPassesTest;
+
+
+@RunWith(Suite.class)
+@SuiteClasses({
+    AlwaysPassesTest.class,
+	JsonpAppClientTest.class
+})
+/**
+ * Purpose: This suite collects and runs all known good test suites.
+ */
+public class FATSuite {
```

Comment:
Lets use a `RepeatTest` classrule here to repeat these tests again with EE8 features.

Reply:
This pull was only intended to cover the port of existing jsonp-1.0 client container FATs to Liberty, and duplicating for jsonp-1.1 was going to be a separate pull, but as long as I'm making other review updates, I'll go ahead and cover it here.